openedx/features/custom_fields/multiselect_with_other/db/fields.py

```diff
@@ -93,8 +93,11 @@ def to_python(self, value):
             if isinstance(value, list):
                 return value
             elif isinstance(value, string_type):
-                choices_str = value.replace(self.other_delimiter, '')
-                selected_choices = [choice for choice in choices_str.split(',') if choice.strip()]
+                choices_before_and_after_delimiter = value.split(self.other_delimiter)
+                selected_choices = [choice for choice in choices_before_and_after_delimiter[0].split(',')
+                                    if choice.strip()]
+                if len(choices_before_and_after_delimiter) > 1:
+                    selected_choices.append(choices_before_and_after_delimiter[1])
```

Comment:
Isn't this change already done in LP-2165?

Reply:
I took a branch from master and this commit is there. (its also verified) Not sure why it is here but I only have 1 commit in this.